.github/workflows/build_and_test_android.yml

```diff
@@ -106,14 +106,17 @@ jobs:
 
   test:
     # These physical devices are not scalable. Only run on postsubmit for now.
-    if: (! inputs.is-pr)
+    # if: (! inputs.is-pr)
     needs: cross_compile
     strategy:
       matrix:
-        # TODO(#9855): Add pixel-6-pro and moto-edge-x30.
         target:
           - device-name: pixel-4
-            label-exclude: vulkan
+            label-exclude: "vulkan"
+          - device-name: pixel-6-pro
+            label-exclude: "^requires-gpu-nvidia$"
```

Comment:
pixel 4 takes ~6 minutes to run tests:
https://github.com/openxla/iree/actions/runs/4927131607/jobs/8804013809?pr=13441
pixel 6 pro takes ~7 minutes to run tests:
https://github.com/openxla/iree/actions/runs/4927131607/jobs/8804014054?pr=13441

Looks like the extra time is coming from Vulkan tests. Could maybe add a comment to the `label-exclude: "vulkan"` for pixel 4 that it has older GPU drivers without support for Vulkan 1.2 (assuming that's why Vulkan is being skipped)

Reply:
Yes, that's correct. Added the comment.